Source/Fluxor/DependencyInjection/FluxorOptions.cs

```diff
@@ -53,6 +64,23 @@ public FluxorOptions(IServiceCollection services)
 			return this;
 		}
 
+		/// <summary>
+		/// The Lifecycle that should be used when registering Fluxor features/reducers/effects/middleware</br>
+		/// (default) LifecycleEnum.Scoped = Create a new instance for each new request</br>
+		/// LifecycleEnum.Singleton = Create a new instance on first request and reuse for rest of application lifetime</br>
+		/// </br>
+		/// NOTE: indicating Singleton should be done only for exceptional cases.
+		/// For example, in MAUI/Blazor hybrid applications, the main MAUI application is a different scope then each BlazorWebView component
+		/// and state needs to be shared across all scopes of the application</br>
+		/// </br>
+		/// This value should only be set once during the configuration of Fluxor
+		/// </summary>
+		public FluxorOptions SetRegistrationLifecycle(LifecycleEnum lifecycle)
```

Comment:
I know this was my idea, but now I've seen it I don't like my idea :)

Would you mind changing this to a property `public  FluxorServiceLifetime ServiceLifetime { get; set; }`

Reply:
I go back and forth...

If it gets changed to a property, then configuration is no longer fluent.

```
			services.AddFluxor(o => o
				.SetStoreLifetime(StoreLifetime.Scoped)
				.ScanAssemblies(typeof(Program).Assembly));
```

versus

```
			services.AddFluxor(o =>
         {
				o.StoreLifetime = StoreLifetime.Scoped;
				o.ScanAssemblies(typeof(Program).Assembly);
         });
```

Reply:
Aha, how about WithServiceLifetime()?

Then I won't get the impression it can change between each ScanAssemblies.

Reply:
done